test/meuse_comp.jl

```diff
@@ -0,0 +1,54 @@
+@testset "Meuse R gstat compatibility" begin
+    """meuse_grid.csv constains (x, y) plus estimation/variance of log(zinc)
+    for global and local (25 samples)
+    """
+    datadir = joinpath(@__DIR__,"data")
+    data_meuse = readgeotable(joinpath(datadir,"meuse.csv"), coordnames=(:x,:y))
+    grid_meuse = readgeotable(joinpath(datadir,"meuse_grid.csv"), coordnames=(:x,:y))
+
+    dim = 2
+    tol = 0.001
+    knn = 25
```

Comment:
The same for knn, I would just pass `maxneighbors=25` directly without introducing a variable.